docs/eventing/samples/iot-core/trigger.yaml

```diff
@@ -8,15 +8,15 @@ spec:
     ref:
       apiVersion: serving.knative.dev/v1alpha1
       kind: Service
-      name: message-dumper
+      name: event-display
 
 ---
 # This is a very simple Knative Service that writes the input request to its log.
 
 apiVersion: serving.knative.dev/v1alpha1
 kind: Service
 metadata:
-  name: message-dumper
+  name: event-display
```

Comment:
Once again the readme will have to be updated. s/message-dumper/event-display/g.